README.md

```diff
@@ -1,41 +1,37 @@
-<img src="docs/images/notary-blk.svg" alt="Notary" width="400px"/>
-
 [![GoDoc](https://godoc.org/github.com/theupdateframework/notary?status.svg)](https://godoc.org/github.com/theupdateframework/notary)
 [![Circle CI](https://circleci.com/gh/theupdateframework/notary/tree/master.svg?style=shield)](https://circleci.com/gh/theupdateframework/notary/tree/master) [![CodeCov](https://codecov.io/github/theupdateframework/notary/coverage.svg?branch=master)](https://codecov.io/github/theupdateframework/notary) [![GoReportCard](https://goreportcard.com/badge/theupdateframework/notary)](https://goreportcard.com/report/github.com/theupdateframework/notary)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Ftheupdateframework%2Fnotary.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Ftheupdateframework%2Fnotary?ref=badge_shield)
 
 # Notice
 
-The Notary project has officially been accepted in to the Cloud Native Computing Foundation (CNCF).
-It has moved to https://github.com/theupdateframework/notary. Any downstream consumers should update
-their Go imports to use this new location, which will be the canonical location going forward.
-
-We have moved the repo in GitHub, which will allow existing importers to continue using the old
-location via GitHub's redirect.
+This repository provides an implementaiton of 
+*[The Update Framework specficaitions](https://github.com/theupdateframework/specification)* 
+and all references to `notary` in this repository refer to the implementation of the client 
+and server aligning with the [TUF](https://github.com/theupdateframework/specification) specification.
```

Comment:
I think it will help if we say 
- the year  ( was it 2016 ?) when notary was released, as that will help readers date it. 
- explicitly state that it does not follow the specifications called out in notaryproject/notaryproject repo. ( I think we are planning to archive this repo, and when we do, we can come optionally come back and change this)

Reply:
+1
Though, I am not sure what we mean with "released" in this case - there has never been 1.0.0 of the implementations. The last stable version is 0.6.1 from 2018 (i.e. 5 years ago), the first implementation was released in Nov 2015 (v0.1)

Reply:
Need to add information for DCT and call out the first release as a follow up.